src/scripts/h5p-multi-media-choice.js

```diff
@@ -290,6 +290,6 @@ export default class MultiMediaChoice extends H5P.Question {
    * Retrieve title of the content type
    */
   getTitle() {
-    return H5P.createTitle((this.contentData && this.contentData.metadata && this.contentData.metadata.title) ? this.contentData.metadata.title : 'Image Choice');
+    return H5P.createTitle((this.extras && this.extras.metadata && this.extras.metadata.title) ? this.extras.metadata.title : 'Image Choice');
```

Comment:
@j0kerZ @thomasmars 
What about `return H5P.createTitle(this.extras?.metadata?.title || 'Image Choice');`?

Optional chaining really makes code easier to understand in my opinion, adoption is pretty good (https://caniuse.com/?search=optional%20chaining) and Babel could even take care of that if necessary.